src/PriceCalculatorKata/Utils/TotalCostCalculation.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace PriceCalculatorKata
+{
+    class TotalCostCalculation
+    {
+        public double Price { get; set; }
+        public double Tax { get; set; }
+        public double TotalDiscount { get; set; }
+        public double AdditionalCosts { get; set; }
+        public double GetTotalCost()
```

Comment:
if this is a utiltiy class, then we should move out the props to a separate model class, and leave this method only in the utility class. You can do an extension file for TotalCostCalculation

Reply:
done and pushed